hnn_core/optimization.py

```diff
@@ -290,15 +290,11 @@ def _optrun(new_params, opt_params, params, opt_dpls, scale_factor,
                      tstop=opt_params['opt_end'],
                      weights=opt_params['weights'])
 
-    if avg_rmse < opt_params['stepminopterr']:
-        best = "[best] "
-        opt_params['stepminopterr'] = avg_rmse
-        opt_dpls['best_dpl'] = avg_dpl
-    else:
-        best = ""
```

Comment:
This created a discrepancy between what `fmin_cobyla()` was returning as the "optimized" parameters and what was being recorded in `opt_params['stepminopterr']` and `opt_dpls['best_dpl']` as the current RMSE and best-fitting dipole, respectively. I'm guessing this was an artifact of a previous optimization method we tried that post-hoc evaluated each of the parameter configuration steps it had tested and selected the best fitting one.